torch/_inductor/codegen/triton_foreach.py

```diff
@@ -6,7 +6,7 @@
 from .common import IndentedBuffer, Kernel
 from .triton import TritonKernel
 from .triton_utils import config_of, signature_of
-
+from typing import List, Any
```

Comment:
This should be a list of Nodes, not a list of Any, right?

Reply:
mind fixing this ?

Reply:
Yeah. Will fix it @eellison